eng/PublishSourceBuild.props

```diff
@@ -80,14 +82,21 @@
     </PropertyGroup>
   </Target>
 
+  <PropertyGroup>
+    <SharedComponentFilterMode>NonToolingOnly</SharedComponentFilterMode>
```

Comment:
The default filter mode is changing from `NonToolingAndBootstrapOnly`. Presumably this is expected?